File: backend/src/main/java/com/back/domain/adoption/dto/request/AdoptionCareStatusUpdateRequestDto.java
package com.back.domain.adoption.dto.request;

import jakarta.validation.constraints.NotNull;

public record AdoptionCareStatusUpdateRequestDto(
        @NotNull
        Long id,

        @NotNull
        String type, // "ADOPTION" 또는 "CARE"

        @NotNull
        String status // "ACCEPTED", "REJECTED"
) {
}

File: backend/src/main/java/com/back/domain/adoption/dto/request/AdoptionCareStatusUpdateRequestDto.kt
package com.back.domain.adoption.dto.request

import jakarta.validation.constraints.NotNull

data class AdoptionCareStatusUpdateRequestDto(
    val id: @NotNull Long = 0L,
    val type: @NotNull String = "",  // "ADOPTION" 또는 "CARE"
    val status: @NotNull String = "" // "ACCEPTED", "REJECTED"
)
